.circleci/config.yml

```diff
@@ -0,0 +1,60 @@
+version: 2.1
+parameters:
+  pixel-driver-version:
+    type: string
+    default: "0.0.1"
+orbs:
+  docker: circleci/docker@1.7.0
+executors:
+  docker-publisher:
+    docker:
+      - image: circleci/buildpack-deps:latest-alpine
+commands:
+  move-src:
+    steps:
+      - run:
+          name:  move src into project folder
+          command: |
+            mkdir esp_pixel_driver
+            cp *.* esp_pixel_driver/
+            cd esp_pixel_driver
+  deps-arduino:
+    steps:
+      - run:
+          name:  install arduino deps
+          command: |
+            arduino-cli lib install "ArtnetWifi"
+            arduino-cli lib install "FastLED"
+            arduino-cli lib install "WiFiManager"
+            arduino-cli lib install "ArduinoJson"
+
+jobs:
+  build:
+    docker:
+      - image: rnavt/esp32builder:1ff8208
```

Comment:
this docker image is defined multiple times. to reduce duplicate hard coding itd be good to put the image into the executors list at the top